packages/mds-stream/kafka/stream-producer.ts

```diff
@@ -28,6 +27,8 @@ import { getKafkaBrokers } from './helpers'
  */
 type RequiredStringKeys<T extends object> = RequiredKeys<T> & ExtendedKeys<T, string>
 
+const BATCH_SIZE = Number(process.env.BATCH_SIZE ?? 1_000)
```

Comment:
It might make sense to call this something a little more verbose (I could imagine a lot of other cases where we might wanna configure sizing for batches of things).

Maybe `KAFKA_PRODUCER_BATCH_SIZE`?

Reply:
Ooh also, there's a pretty slick library called `envalid` that could be used for the env var validation/defaulting if you want too!

Reply:
Wow. This is the first time you've ever asked me to make something more verbose. Screenshotting this.